src/components/cabinet/qrcode-order-modal.component.js

```diff
@@ -0,0 +1,174 @@
+import React, { useRef, forwardRef, useState, setState } from "react";
+import Modal from "react-modal";
+import { CrossIcon, EmailIcon, CopyIcon } from "icons";
+import { getProfileQrCodeImage } from "utils/profile.utils";
+import { uploadFile } from "services/api/profile.service";
+import Counter from 'components/counter/counter.component';
```

Comment:
Our project is built in such a way that there is no point in referring to the full file path when importing a Counter component, so we will do with components/counter
```suggestion
import Counter from 'components/counter';
```